org/postgresql/jdbc4/AbstractJdbc4Connection.java

```diff
@@ -99,7 +99,7 @@ public Struct createStruct(String typeName, Object[] attributes) throws SQLExcep
     public Array createArrayOf(String typeName, Object[] elements) throws SQLException
     {
         checkClosed();
-        int oid = getTypeInfo().getPGArrayType(typeName);
+        int oid = getTypeInfo().getPGArrayType(typeName.toLowerCase());
```

Comment:
I did not notice you create PRs from your own repository.
Then you should push to your repo.